format/src/main/thrift/schema.thrift

```diff
@@ -124,6 +124,7 @@ struct TableSchema{
 	1: required string table_id;  // ID used to
 	2: required list<ColumnSchema> table_columns; // Columns in the table
 	3: required SchemaEvolution schema_evolution; // History of schema evolution of this table
+	4: optional i32 block_size
```

Comment:
I think it is better to save it in the table metadata, through metastore for example. 
By concept, this should not be tied with the file.
@ravipesala @gvramana any suggestion?